### PR TITLE
Fix for: ignores backend settings #904

### PR DESCRIPTION
**Pull request by BarryCarlyon**
The code here is doing a boolean check but the setting is saved as a string (no, yes or yes_admin)
As per #904 this fixes the config geneator

### CODE REVIEW

**Comment by BarryCarlyon** [comment]
There we go, got my branches in sync properly now.
Is it home time yet? (10am Friday morning…)

File: app/code/community/Nexcessnet/Turpentine/Model/Varnish/Configurator/Abstract.php [hunk 1]
```diff
@@ -292,7 +292,7 @@ protected function _getDefaultBackend() {
             'first_byte_timeout'    => $timeout . 's',
             'between_bytes_timeout' => $timeout . 's',
         );
-        if ( Mage::getStoreConfigFlag( 'turpentine_vcl/backend/load_balancing' ) ) {
+        if ( Mage::getStoreConfigFlag( 'turpentine_vcl/backend/load_balancing' ) != 'no' ) {
```
Comment (miguelbalparda) [inline app/code/community/Nexcessnet/Turpentine/Model/Varnish/Configurator/Abstract.php]: This seems to be the only relevant code change, the other ones seems to be indentation and style. Can you confirm @BarryCarlyon ?

Comment (BarryCarlyon) [inline app/code/community/Nexcessnet/Turpentine/Model/Varnish/Configurator/Abstract.php]: Yip.

My text editor wiped out some unneeded white space

Comment (BarryCarlyon) [inline app/code/community/Nexcessnet/Turpentine/Model/Varnish/Configurator/Abstract.php]: And converted some TAB to 4 Space Tabs (like the rest of the file/consistency)

**Comment by miguelbalparda** [comment]
Thank you for your contribution @BarryCarlyon!

**Comment by BarryCarlyon** [comment]
*tips hat*

No problem!

**Comment by craigcarnell** [comment]
@BarryCarlyon Mine is saved as a boolean.. any ideas why?

**Pull request merged**